### ISSUE TITLE
Update kms docker to use distroless

### ISSUE THREAD

**Issue by giuseppe-sky** [sky-uk/kafka-message-scheduler#96]
Removed

**Comment by r-glyde** [comment]
Hi @giuseppe-sky. Could you please provide a bit more context for this? Have you been told that any service you are using must use these base images? This project has been open sourced I believe so is available to non-sky users.

We'll need to have a bit of a look but initial feeling is that open projects should be using open base images.

**Comment by lacarvalho91** [comment]
Agreed with @r-glyde, the issue description doesn't describe anything.
